tools/management/commands/upload_excel_bias.py

```diff
@@ -471,57 +338,56 @@ def fetch_vendor(self, ligand,experiment_entry):
                 ligand_vendor = BiasedExperimentVendors(experiment=experiment_entry,
                                                         vendor=x)
                 ligand_vendor.save()
-        self.logger.info("ligand_vendor saved")
 
-    def fetch_protein(self,protein_from_excel, source):
+
+    @staticmethod
+    def fetch_protein(protein_from_excel):
         """
         fetch receptor with Protein model
         requires: protein id, source
         """
-        test = None
-        if Protein.objects.filter(entry_name=protein_from_excel):
-            protein = Protein.objects.filter(entry_name=protein_from_excel)
-            test = protein.get()
-        elif Protein.objects.filter(web_links__index=protein_from_excel, web_links__web_resource__slug='uniprot'):
-            protein1 = Protein.objects.filter(
-                web_links__index=protein_from_excel, web_links__web_resource__slug='uniprot')
-            test = protein1[0]
-        if test == None:
-            self.logger.info("fetch_protein  error")
-        return test
-
-    def fetch_ligand(self, ligand_id, ligand_type, ligand_name, source_file):
+        try:
+            test = None
+            if Protein.objects.filter(entry_name=protein_from_excel):
+                protein = Protein.objects.filter(entry_name=protein_from_excel)
+                test = protein.get()
+            elif Protein.objects.filter(web_links__index=protein_from_excel, web_links__web_resource__slug='uniprot'):
+                protein1 = Protein.objects.filter(
+                    web_links__index=protein_from_excel, web_links__web_resource__slug='uniprot')
+                test = protein1[0]
+            return test
+        except:
+            import pdb; pdb.set_trace()
```

Comment:
Can we remove this try/except that leads to an open pdb tracing in the shell?

Reply:
This one is fixed